docs/root/operations/admin.rst

```diff
@@ -141,3 +141,8 @@ The fields are:
   .. http:get:: /stats?format=json
 
   Outputs /stats in JSON format. This can be used for programmatic access of stats.
+
+  .. http:get:: /stats?format=prometheus
+
+  Outputs /stats in `Prometheus <https://prometheus.io/docs/instrumenting/exposition_formats/>`_
```

Comment:
Can you add some text here that roughly says only counters/gauges are output currently and that histograms will be output in a future update?

Reply:
🆗